client/pages/custCart.jsx

```diff
@@ -39,7 +40,21 @@ export default class AddCategory extends React.Component {
 
   handleSubmit(event) {
     event.preventDefault();
-
+    const req = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        userId: this.context.user.userId,
+        custNote: this.state.custNote
+      })
```

Comment:
Ok, you will need to start sending the X-Access-Token now instead of the userId in the body. Your server will need to use authorizationMiddleware to get the userId from the token instead of trusting the client.